python/pyproject.toml

```diff
@@ -68,6 +67,7 @@ python = [
     "confluent-kafka<=2.11.1",
     "fastavro>=1.4.11,<=1.11.1",
     "tqdm",
+    "hops-deltalake; python_version >= '3.9' and (sys_platform == 'linux' or (sys_platform == 'darwin' and platform_machine == 'arm64'))",
```

Comment:
do your tests work when using python 3.8?

Reply:
I'll raise it in slack and can do the necessary changes tomorrow